moduleframework/tools/modulelint.py

```diff
@@ -27,8 +27,8 @@
 
 class ModuleLintSigning(module_framework.AvocadoTest):
     """
-    :avocado: disable
-    :avocado: tags=WIP
+    :avocado: enable
+    :avocado: tags=docker,fedora,rhel,tier1,WIP
```

Comment:
We should think how to structure tests. Some tests are mandatory and some tests are optional.
This should be discussed with other folks like Alex Corwin.

Reply:
I agree that this is the most important feature for me. Some container owners are going to have valid reasons to want to disable a specific test. Ideally, that container owner could keep some sort of metadata file alongside their container/dockerfile that would specify which tests to enable/disable. I'm thinking something similar to the pylintrc for the pylint python linter. When you run the pylint tool, it knows to look for a pylintrc file that controls what tests to skip.

Reply:
Yep, we have to think about that more. what it the best way how to do that. This PR uses avocado tags, as main filter possibilities, so everything is possible to filter from `cmdline`, but maybe we can define it also in some file what `mtf` tool will use, instead of `-t` param from cmdline. I can imagine something like file `.mtfrc.yaml` with content like:
```
module_lint_tags:
     - sanity,-optional
     - composecheck
```
what will be transformed to `mtf` args `-t sanity,-optional -t composecheck`. Does it make sense?

Reply:
Yes, I think that sounds perfect.

Reply:
.pylintrc has a INI file like format. Do you need really YAML structure?
Based on the output from `pylint --generate-rcfile`

Reply:
Nope, I don't think the exact format matters